simulator/src/main/java/com/google/cloud/App.java

```diff
@@ -177,14 +194,13 @@ Iterator<List<OrderBookEvent>> buildSimulator() {
    */
   MatcherContext buildMatcherContext() throws ParameterException {
     String sessionId = DateTimeFormatter.ofPattern("yyyy-MM-dd.HH:mm").format(LocalDateTime.now());
-    System.out.println("Session id for this simulation: " + sessionId);
```

Comment:
This output is useful for my tests and is referenced in README. Please make it conditional based on the type of the output.

Reply:
Added back. I removed that temporarily as part of taking the JSON output -- loading JSON into BigQuery (at a huge scale with gzip'd files) can be slow. Avro is meaningfully faster.